Nos/Service/DependencyInjection.swift

```diff
@@ -187,3 +192,9 @@ fileprivate enum MediaServiceKey: DependencyKey {
     static let testValue: any MediaService = MockMediaService()
     static let previewValue: any MediaService = DefaultMediaService() // enables us to manually test with previews
 }
+
+fileprivate enum PreviewEventRepositoryKey: DependencyKey {
+    static let liveValue: any PreviewEventRepository = DefaultPreviewEventRepository()
+    static let testValue: any PreviewEventRepository = DefaultPreviewEventRepository()
+    static let previewValue: any PreviewEventRepository = DefaultPreviewEventRepository()
```

Comment:
I don't think `testValue` and `previewValue` are required, and `Dependencies` will just use the `liveValue` if they're not specified. But they don't hurt anything, so no big deal -- just wanted to make you aware.